packages/generator/src/sdk-metadata/code-samples.ts

```diff
@@ -29,3 +30,47 @@ export function genericCodeSample(): InstructionWithTextAndHeader {
     header: usageHeaderText
   };
 }
+
+export function importsCodeSample(
```

Comment:
Could you export two methods functionImportCodeSample and actionImportCodeSample which delegate to this private method.